CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## UNRELEASED
 
+### Changed
+
+- Update eslint-config-airbnb to 18.0.1
+  - Migration guide here [v5 to v6](./docs/migration-from-5-to-6.md)
```

Comment:
Given that this requires a migration guide, maybe it's breaking?

Reply:
yep, this should be a breaking change.

Reply:
Yep in the `package.json` its set as Breaking but you are right this should be changed to say Breaking

Reply:
I think in the past historically Changed has been used in favour of Breaking as its not listed in all the history but its our standard so will change to align

Reply:
Addressed